src/components/ui/Image/Image.tsx

```diff
@@ -1,23 +1,55 @@
 import { useGetThumborImageData } from '@vtex/gatsby-plugin-thumbor'
 import { GatsbyImage } from 'gatsby-plugin-image'
 import React, { useMemo } from 'react'
-import imagesConf from 'src/images/config'
+import type { ThumborImageOptions } from '@vtex/gatsby-plugin-thumbor'
 import type { GatsbyImageProps } from 'gatsby-plugin-image'
 
-interface Props extends Omit<GatsbyImageProps, 'image'> {
-  variant: string
-  src: string
-}
+interface Props extends Omit<GatsbyImageProps, 'image'>, ThumborImageOptions {}
```

Comment:
What do you think about making this `Props` interface so it better separates which properties are from `Thumbor` and which are from `GatsbyImage`? This way it might be easier to fiddle with this component later if someone decides to use an image service other than `Thumbor` or if we do not use `GatsbyImage` anymore.

Maybe something like this?
```TypeScript
interface Props extends Omit<GatsbyImageProps, 'image'> {
  thumborOptions: ThumborImageOptions
}
```

And then we could call the `Image` component like:
```jsx
// I don't know exactly which attributes are from GatsbyImage only
<Image baseUrl={...} /*  other GatsbyImage/img attributes here */ thumborOptions={{...}} />
```

Reply:
I'm not sure. This would case using the component a little bit weird, because this `baseURL` should go inside `thumborOptions`, even though this baseURL is used by GatsbyImage hook and not by thumbor itself. Look, an example:
```tsx
// How it's with current implemetation:
<Image
  baseUrl={productImages[0].url}
  sourceWidth={720}
  aspectRatio={1}
  width={720}
  breakpoints={[250, 360, 480, 720]}
  layout="constrained"
  backgroundColor="#f0f0f0"
  options={{
    fitIn: true,
  }}
  alt={productImages[0].alternateName}
  loading="eager"
/>

// How it will be after the suggested changes
<Image
  thumborOptions={{
    baseUrl: productImages[0].url,
    sourceWidth: 720,
    aspectRatio: 1,
    width: 720,
    breakpoints: [250, 360, 480, 720],
    layout: "constrained",
    backgroundColor: "#f0f0f0",
    options: {
      fitIn: true,
    },
  }}
  alt={productImages[0].alternateName}
  loading="eager"
/>
```